Dockerfile

```diff
@@ -4,19 +4,29 @@
 #       Changes to this file must be copied over to the other file.
 
 ARG GUI=without
+ARG MYLDFLAGS="-s -Wl,--no-undefined -lblas"
+ARG MYCFLAGS="-O2 -std=gnu99 -m64"
+ARG MYCXXFLAGS=""
```

Comment:
Is it a convention to use these variable names?

Reply:
No, they have just been copied from my old compile script from decades ago and carried around since then :-D
Feel free to change the variable names.